code/modules/vending/youtool.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "Tools for tools."
 	icon_state = "tool"
 	icon_deny = "tool-deny"
-	//req_access_txt = "12" //Maintenance access
+	//req_access = list(ACCESS_MAINT_TUNNELS)
```

Comment:
This access req was commented out to begin with. No idea why.

Reply:
remove it

Reply:
Remove the req or that it's commented out?

Reply:
remove the enitre line

Reply:
Gotcha